.github/CODEOWNERS

```diff
@@ -84,6 +84,7 @@ llvm/**/append-file/ @mdtoguchi @AGindinson
 llvm/**/file-table-tform/ @mlychkov @AlexeySachkov @kbobrovs
 llvm/**/llvm-foreach/ @AlexeySachkov  @Fznamznon
 llvm/**/llvm-no-spir-kernel/ @AGindinson @AlexeySachkov
+llvm/**/spir-to-ir-wrapper/ @mdtoguchi @AGindinson
```

Comment:
It should probably become @intel/dpcpp-clang-driver-reviewers by the time the patch is ready for merge. Can be edited right now or fixed when resolving the conflicts against https://github.com/intel/llvm/pull/5016 (the latter will probably be needed anyway).

Reply:
I'll adjust with the conflict resolution.